app/generated_resources_override.grd

```diff
@@ -827,7 +827,7 @@ are declared in tools/grit/grit_rule.gni.
           </if>
       </if> <if expr="is_chromeos">
         </if> <if expr="not is_android">
-        <message name="IDS_EXTENSIONS_INSTALL_LOCATION_UNKNOWN" desc="The text explaining the the installation location is unknown.">
+        <message name="IDS_EXTENSIONS_INSTALL_LOCATION_UNKNOWN" desc="The text explaining the installation location is unknown.">
```

Comment:
This file is auto-generated. This type would need to be fixed in Chromium code (https://source.chromium.org/chromium/chromium/src/+/main:chrome/app/generated_resources.grd;l=10681)